.changeset/two-coins-compare.md

```diff
@@ -0,0 +1,5 @@
+---
+"wrangler": patch
+---
+
+Disabled local persistence by default & added --experimental-enable-local-persistence flag
```

Comment:
Oh last thing... Could you add a breaking changes notification here... Something like:

```
BREAKING CHANGE:

When running `dev` locally any data stored in KV, Durable Objects or the cache are no longer persisted between sessions by default.

To turn this back on add the `--experimental-enable-local-persistence` at the command line.
```

Reply:
Like this?